src/main/java/com/mixpanel/android/mpmetrics/MixpanelWebViewActivity.java

```diff
@@ -1,6 +1,36 @@
 package com.mixpanel.android.mpmetrics;
 
 import android.app.Activity;
+import android.os.Bundle;
+import android.webkit.WebChromeClient;
+import android.webkit.WebSettings;
+import android.webkit.WebView;
+import android.webkit.WebViewClient;
+
+import com.mixpanel.android.R;
 
 public class MixpanelWebViewActivity extends Activity {
```

Comment:
why did this class already exist? what was it used for?

Reply:
It didn't, I think the class creation is either in another PR or wasn't tracked

Reply:
ah, it should be in this one though shouldn't it? along with whatever code in the router that calls it?

Reply:
yeah found it on other PR going to fix up